Manage the installation of packages with full version capabilities support. (#879)
STORM-IRIT/Radium-Engine: tests/ExampleApps/HeadlessLibAndApp/libHeadless/CMakeLists.txt

    cmake_policy(SET CMP0042 NEW)
endif(APPLE)

project(RadiumHeadless VERSION 1.0.2)

> # Defines the RadiumHeadless version in the parent scope so that on can use this lib doing
> # add_subdirectory(...) and access to the

Reviewer: ```suggestion
# Defines the RadiumHeadless version in the parent scope so that one can use this lib doing
# add_subdirectory(...) and access RadiumHeadless_VERSION
```
